packages/comps/src/utils/derived-nfl-dailies.ts

```diff
@@ -86,7 +117,29 @@ export const getMarketTitle = (
     title = marketTitles.title;
     description = populateHomeAway(marketTitles.description, homeTeam, awayTeam);
   }
+  if (sportsMarketType === 1) {
+    // spread
+    let fav = awayTeam;
+    let underdog = homeTeam;
+    if (Number(line) < 0) {
+      underdog = awayTeam;
+      fav = homeTeam;
+    }
+    let spread = new BN(line).abs().toNumber();
+    if (!Number.isInteger(spread)) {
+      spread = Math.trunc(spread);
+    }
+    title = marketTitles.title
+      .replace(NAMING_TEAM.FAV_TEAM, fav)
+      .replace(NAMING_TEAM.UNDERDOG_TEAM, underdog)
+      .replace(NAMING_LINE.SPREAD_LINE, String(spread));
+  }
 
+  if (sportsMarketType === 2) {
```

Comment:
also

Reply:
Matches the pattern of the other utils. Will need to be addressed in another refactor.